src/main/resources/assets/immersiveengineering/manual/es_es/es_es.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
why was this added?

Reply:
> why was this added?
I'm about to translate the engineer manual completely in Spanish, and I put that just to remember it, sorry if it bothers there

Reply:
> why was this added?

Tomorrow I will finish everything and update the PR

Reply:
0.1% to complete. I worked hard on it today. In a few hours I'll give it a final review and let you know when it's finally ready. I just hope they don't update before so I don't get the translation left out :( Let me know if they are about to release it please. Regards!

Reply:
> why was this added?

Hi! Wow, this was a colossal job, it took me much longer than I thought it would. I have done a final check of all the files and strings and fixed a lot of inconsistencies everywhere. I'm done with the translation. I like to do things right and maybe I'm too detail oriented, but in the end it's a quality job.
Whenever you want you can merge the changes.
Thanks for your patience and the truth is that the pull request is full of commits, I don't know if it's the cleanest way to do it but let me know how I can improve. Thanks and greetings! Nothing more to say.